sdks/python/apache_beam/options/pipeline_options.py

```diff
@@ -714,6 +714,18 @@ def _add_argparse_args(cls, parser):
          'job submission, the files will be staged in the staging area '
          '(--staging_location option) and the workers will install them in '
          'same order they were specified on the command line.'))
+    parser.add_argument(
+        '--jar_package', '--jar_packages',
+        dest='jar_packages',
+        action='append',
+        default=None,
```

Comment:
This should be another separate JIRA and a PR (specifying Java dependencies from Python).

Reply:
Thanks for the comment. Will do.